docs/reddit-migration/considerations/sorting.md

```diff
@@ -0,0 +1,32 @@
+### Sorting
+
+There are several areas where reddit sorts objects coming back from their APIs in a way that isn't necessarily trivial via standard django ORM methods. This list is meant to be comprehensive so we know the problem space:
```

Comment:
Are we generally happy with reddit's sorting or could this be an opportunity to come up with something different instead of trying to match its algorithms?

Reply:
I'm not opposed to it, but I think we should be careful about what changes could impact the UI and for me that would be the line of whether we do it as part of the migration or as a subsequent step.

That said, keeping behavior as similar as possible is probably going to make verifying the switchover easier on ourselves, so there's that to consider.